jobsdb/integration_test.go

```diff
@@ -575,8 +577,11 @@ func TestJobsDB(t *testing.T) {
 			TriggerMigrateDS: func() <-chan time.Time {
 				return triggerMigrateDS
 			},
+			configGetter: c,
 		}
 		prefix := strings.ToLower(rand.String(5))
+		t.Setenv(config.ConfigKeyToEnv(config.DefaultEnvPrefix, "JobsDB.jobDoneMigrateThreshold"), "0.7")
+		t.Setenv(config.ConfigKeyToEnv(config.DefaultEnvPrefix, "JobsDB.jobMinRowsMigrateThreshold"), "0.6")
```

Comment:
env variables are global, better set the proper configuration parameters in the config instance itself
```suggestion
		c.Set("JobsDB.jobDoneMigrateThreshold", 0.7)
		c.Set("JobsDB.jobMinRowsMigrateThreshold", 0.6)
```